doc/man1/openssl-s_client.pod.in

```diff
@@ -316,39 +319,63 @@ abort the handshake with a fatal error.
 
 Limit verify output to only errors.
 
+=item B<-CAfile> I<file>
+
+A file in PEM format containing trusted CA certificates.
```

Comment:
concatenated
potentially with some extra information

Reply:
Added 'one or more', also for `-verifyCAfile` and `-chainCAfile`.
I did not add 'concatenated' because this may be misunderstood that there must not be anything between certificate entries (while in fact there may be interspersed text, CRLs, etc.).

> potentially with some extra information

Good point!
Though for PEM files the use of concatenation for including multiple entries should well-known,
I found that for `-CApath` the situation is different: each file in the referenced directory is assumed to contain just one cert.
Looking deeper into this, I found and fixed further doc issues regarding the `rehash` app.